### PR TITLE
Java: Adding command MSetNX

### PR DESCRIPTION
**Pull request by tjzhang-BQ**

### CODE REVIEW

File: java/client/src/main/java/glide/api/commands/StringBaseCommands.java [hunk 1]
```diff
@@ -156,6 +156,25 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> mset(Map<String, String> keyValueMap);
 
+    /**
+     * Sets multiple keys to multiple values in a single operation.Perform any operation at all even
+     * if just a single key already exists.
```
[the remark this answers is not in the document]
Comment (tjzhang-BQ) [reply java/client/src/main/java/glide/api/commands/StringBaseCommands.java]: just noticed this too right after pushing, sloppy copy

File: java/client/src/main/java/glide/api/commands/StringBaseCommands.java [hunk 2]
```diff
@@ -156,6 +156,25 @@ public interface StringBaseCommands {
...
+     *
+     * @apiNote When in cluster mode, the command may route to multiple nodes when keys in <code>
+     *     keyValueMap</code> map to different hash slots.
```
[the remark this answers is not in the document]
Comment (tjzhang-BQ) [reply java/client/src/main/java/glide/api/commands/StringBaseCommands.java:164]: Does the test added in `CommandTest` require the redis-rs patch to pass?

[the remark this answers is not in the document]
Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/StringBaseCommands.java:164]: Yes

[the remark this answers is not in the document]
Comment (Yury-Fridlyand) [reply java/client/src/main/java/glide/api/commands/StringBaseCommands.java:164]: Please change apinote and add correspodning test

**Pull request merged**